lib/api.js

```diff
@@ -1981,8 +1982,20 @@ module.exports = function(self, options) {
   // You can force an export to a locale that does not exist yet
   // for the doc in question. In this situation the doc is replicated
   // to the new locale.
-
-  self.forceExport = function(req, id, locales, callback) {
+  //
+  // `options` may be completely omitted. If `options.existing` is
+  // true or `options` is omitted, the document is exported whether
+  // it already exists in each locale or not. If `options.existing`
+  // is false, it is exported to each locale only if it does not
+  // already exist there.
+
+  self.forceExport = function(req, id, locales, options, callback) {
+    if (!callback) {
+      callback = options;
```

Comment:
if `options` is undefined, `callback` will be undefined as well

Reply:
callback is always present, so if it is falsy that means it's really just a call without an options argument. So I'm correctly shuffling the fourth argument to the fifth to account for it.